dask/dataframe/core.py

```diff
@@ -5055,7 +5058,20 @@ def eval(self, expr, inplace=None, **kwargs):
         return self.map_partitions(M.eval, expr, meta=meta, inplace=inplace, **kwargs)
 
     @derived_from(pd.DataFrame)
-    def dropna(self, how="any", subset=None, thresh=None):
+    def dropna(self, how=pd_no_default, subset=None, thresh=pd_no_default):
+
+        if (
```

Comment:
This is causing a problem in the dropna test when we do `assert_eq(ddf.dropna(), df.dropna())` I think it's because it's trying to use pandas defaults. 

I'm not quite sure how to make things work on the tests to be consistent.